tests/publiccloud/registercloudguest.pm

```diff
@@ -0,0 +1,61 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: cloud-regionsrv-client
+# Summary: Test system (re)registration
```

Comment:
I would add here the reference to https://github.com/SUSE-Enceladus/cloud-regionsrv-client/blob/master/integration_test-process.txt or do you think that doesn't make sense? e.g.

    # Summary: Test system registration
    #          See https://github.com/SUSE-Enceladus/cloud-regionsrv-client/blob/master/integration_test-process.txt for the agreed test procedure